priv/repo/seeds.exs

```diff
@@ -93,6 +94,17 @@ demo =  User.changeset(%User{}, %{name: "Demo User",
                          })
         |> Repo.insert!()
 
+admin =  User.changeset(%User{}, %{name: "Facility Admin",
+                          family_members: 0,
+                          primary_language_id: 184,
+                          facility_id: 1,
+                          email: "admin@example.com",
+                          password: "password",
+                          password_confirmation: "password"
```

Comment:
missing a comma at the end here :-)  But I think I'll merge and fix in master, overall looking good!

Reply:
That comma was optional, but it can be added if its better for the project's codestyle. :)

Reply:
Oh, right, didn't see the line under it...
Should be easy to fix in a simple commit.